src/index.js

```diff
@@ -1,18 +1,38 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import { BrowserRouter } from "react-router-dom";
+import { Provider } from "react-redux";
 import "./index.css";
```

Comment:
Import statements seperation. Try to make these things a part of your coding conventions